src/solus-mate-transition-tool.py

```diff
@@ -271,14 +274,14 @@ def on_resolve_async(source, result, data=None) -> bool:
                 name = i.get_id()
                 is_installed = i.get_info() & PackageKitGlib.InfoEnum.INSTALLED == 1
                 not_installed = i.get_info() & PackageKitGlib.InfoEnum.INSTALLED == 0
-                print("is_installed", is_installed, name)
+                print(_("is_installed"), is_installed, name)
```

Comment:
Hey @joebonrichie should the string here be "is installed"? instead?

Reply:
yeah looks like it

Reply:
Changed it